docs/api-reference/next.config.js/environment-variables.md

```diff
@@ -41,7 +41,7 @@ For example, the following line:
 return <h1>The value of customKey is: {process.env.customKey}</h1>
 ```
 
-Will end up being:
+will end up being:
```

Comment:
I'm not sure if this is right. In our curernt style every paragraph always starts with a uppercase.

Reply:
I don't think it's correct to begins every paragraph with an uppercase. The starting paragraph, the code that follows it, and the paragraph that follows the code make a single unit.